src/stage0.txt

```diff
@@ -34,4 +34,4 @@ cargo: 0.29.0
 # looking at a beta source tarball and it's uncommented we'll shortly comment it
 # out.
 
-dev: 1
+#dev: 1
```

Comment:
Not sure if you tested -- we might need to bump the date above to 08-02 as well

Reply:
Oh, woops! Fixed.